jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -84,6 +84,30 @@ final case class ConnectionRetrySettings(connectTimeout: FiniteDuration = 10.sec
   def withMaxBackoff(maxBackoff: Long, unit: TimeUnit): ConnectionRetrySettings =
     copy(maxBackoff = Duration(maxBackoff, unit))
   def withMaxRetries(maxRetries: Int): ConnectionRetrySettings = copy(maxRetries = maxRetries)
+  def withInfiniteRetries(): ConnectionRetrySettings = withMaxRetries(-1)
```

Comment:
Added some helper methods for indefinite retries

Reply:
Nice!